tests/unit/test_match.py

```diff
@@ -88,7 +86,7 @@ def test_match_tlv(self):
         )
 
     def test_match01_as_dict(self) -> None:
-        """Test match01 as_dict."""
+        """Test match04 as_dict."""
```

Comment:
Aldo, could you update the name of the test as `test_match04_as_dict` as well?

Reply:
Yes, I totally missed that.